.github/minty.yaml

```diff
@@ -0,0 +1,21 @@
+version: 'minty.abcxyz.dev/v2'
+
+rule:
+  if: |-
+    assertion.iss == 'https://token.actions.githubusercontent.com' &&
+    assertion.repository_owner_id == '93787867' &&
+    assertion.repository_id == '575621363' &&
+    assertion.ref == 'refs/heads/main'
+
+scope:
+  create-tag:
+    rule:
+      if: |-
+        assertion.job_workflow_ref.startsWith("abcxyz/pkg/.github/workflows/create-tag.yml" &&
```

Comment:
Second thought, i think this should be `        assertion.job_workflow_ref == "abcxyz/pkg/.github/workflows/create-tag.yml@refs/heads/main" &&`